ajax/libs/fast-xml-parser/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "fast-xml-parser",
+  "version": "2.2.1",
```

Comment:
@extend1994 Is it okay to let the bot to update this version for me? I'm not sure.